float64 franka_effector_ddx
float64 franka_effector_ddy
float64 franka_effector_ddz